v21.1/schema-design-schema.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Create a User-defined Schema
+summary: Create a user-defined schema for your CockroachDB cluster
+toc: true
+---
+
+This page provides best-practice guidance on creating user-defined schemas, with a simple example based on Cockroach Labs' fictional vehicle-sharing company, [MovR](movr.html).
+
+{{site.data.alerts.callout_success}}
+For detailed reference documentation on the `CREATE SCHEMA` statement, including additional examples, see the [`CREATE SCHEMA` syntax page](create-schema.html).
+{{site.data.alerts.end}}
+
+<div class="filters filters-big clearfix">
+  <button class="filter-button" data-scope="local">Local</button>
+  <button class="filter-button" data-scope="cockroachcloud">CockroachCloud</button>
+</div>
+
+## Before you begin
+
+Before reading this page, do the following:
+
+<ul>
+  <li>
+    <a href="install-cockroachdb.html">Install CockroachDB.</a>
+  </li>
+  <li>
+    <a class="filter-content" data-scope="local" href="secure-a-cluster.html">Start a local CockroachDB cluster.</a>
+    <a class="filter-content" data-scope="cockroachcloud" href="cockroachcloud/create-your-cluster.html">Create a CockroachCloud cluster.</a>
+  </li>
+  <li>
+    <a href="schema-design-overview.html">Review the database schema objects.</a>
+  </li>
+  <li>
+    <a href="schema-design-database.html">Create a database.</a>
+  </li>
+</ul>
+
+## Create a user-defined schema
+
+User-defined schemas belong to the second level of the [CockroachDB naming hierarchy](sql-name-resolution.html#naming-hierarchy).
+
+To create a user-defined schema, use a [`CREATE SCHEMA` statement](create-schema.html), following [the user-defined schema best practices](#user-defined-schema-best-practices). After reviewing the best practices, see the example we provide [below](#example).
+
+### User-defined schema best practices
+
+Here are some best practices to follow when creating and using user-defined schemas:
+
+- Do not use the preloaded `public` schema. Instead, create a user-defined schema with `CREATE SCHEMA`, and then refer to lower-level objects with fully-qualified names (e.g., `schema_name.table_name`).
```

Comment:
@vy-ton 

Following up on the SQL product area meeting, this is the guidance we currently give re: user-defined schemas. We can remove this recommendation, or we can reword it to recommend that users create UDS if they fit their use case (e.g., different users).

Reply:
I think we should reword. There's no reason to avoid the `public` schema. Database objects can also be moved between schemas as long as there's no name collision. So you can start with `public` and then move to user-defined schemas as needed.

What we want to emphasize is that UDS provides another level that can be secured. And to showcase that with an example like users. Sometimes tests would execute in their own UDS as well